data/strings/sound.txt

```diff
@@ -1,485 +1,485 @@
-    pl = Skręć w prawo.
-    pt = Vire à direita.
-    pt-BR = Vire à direita.
-    ro = Virați la dreapta.
-    ru = Поверните направо.
-    sk = Odbočte doprava.
-    sv = Sväng höger.
-    sw = Zunguka upande wa kulia.
+    en = Turn right
+    ar = انعطف يمينا
+    be = Павярніце направа
+    cs = Odbočte vpravo
+    da = Drej til højre
+    de = Rechts abbiegen
+    el = Στρίψε δεξιά
+    es = Gire a la derecha
+    eu = Biratu eskuinera
+    fa = به راست بپیچید
+    fi = Käänny oikealle
+    fr = Tournez à droite
+    hi = दायीं ओर मुड़ें
+    hr = Skrenite udesno
+    hu = Forduljon jobbra
+    id = Belok kanan
+    it = Gira a destra
+    ja = 右に曲がります
+    ko = 우회전하십시오
+    mr = उजवीकडे वळा
+    nl = Sla rechtsaf
+    pl = Skręć w prawo
+    pt = Vire à direita
+    pt-BR = Vire à direita
+    ro = Virați la dreapta
+    ru = Поверните направо
+    sk = Odbočte doprava
+    sv = Sväng höger
+    sw = Zunguka upande wa kulia
     th = เลี้ยวขวา
-    tr = Sağa dönün.
-    uk = Поверніть праворуч.
-    vi = Rẽ phải.
-    zh-Hans = 右转。
-    zh-Hant = 右轉。
+    tr = Sağa dönün
+    uk = Поверніть праворуч
+    vi = Rẽ phải
+    zh-Hans = 右转
+    zh-Hant = 右轉
 
   [make_a_sharp_right_turn]
-    en = Sharp right.
-    ar = إلى أقصى اليمين.
-    be = Рэзка павярніце направа.
-    cs = Ostře doprava.
-    da = Skarpt til højre.
-    de = Scharf rechts.
-    el = Απότομα δεξιά.
-    es = Gire bruscamente a la derecha.
-    eu = Eman bira itxia eskuinera.
-    fa = منتها الیه سمت راست.
-    fi = Jyrkkä oikea.
-    fr = À droite toute.
-    hi = तीव्रता से दाएं मुड़े।
-    hr = Oštro udesno.
-    hu = Kanyarodjon élesen jobbra.
-    id = Belok tajam ke kanan.
-    it = Svoltare a destra.
-    ja = 急に右折。
-    ko = 급우회전.
+    en = Sharp right
+    ar = إلى أقصى اليمين
+    be = Рэзка павярніце направа
+    cs = Ostře doprava
+    da = Skarpt til højre
+    de = Scharf rechts
+    el = Απότομα δεξιά
+    es = Gire bruscamente a la derecha
+    eu = Eman bira itxia eskuinera
+    fa = منتها الیه سمت راست
+    fi = Jyrkkä oikea
+    fr = À droite toute
+    hi = तीव्रता से दाएं मुड़े
+    hr = Oštro udesno
+    hu = Kanyarodjon élesen jobbra
+    id = Belok tajam ke kanan
+    it = Svoltare a destra
+    ja = 急に右折
+    ko = 급우회전
     mr = तीव्र उजवे वळण
-    nl = Scherp rechtsaf.
-    pl = Ostro w prawo.
-    pt = Curva acentuada à direita.
-    pt-BR = Curva acentuada à direita.
-    ro = Virați strâns la dreapta.
-    ru = Резко поверните направо.
-    sk = Ostro doprava.
-    sv = Skarpt höger.
-    sw = Fanya kulia mkali.
+    nl = Scherp rechtsaf
+    pl = Ostro w prawo
+    pt = Curva acentuada à direita
+    pt-BR = Curva acentuada à direita
+    ro = Virați strâns la dreapta
+    ru = Резко поверните направо
+    sk = Ostro doprava
+    sv = Skarpt höger
+    sw = Fanya kulia mkali
     th = หักมุมขวา
-    tr = Tam sağa dönün.
-    uk = Різко вправо.
-    vi = Rẽ ngoặt bên phải.
-    zh-Hans = 向右急转。
-    zh-Hant = 向右急轉。
+    tr = Tam sağa dönün
```

Comment:
It should be easy to programmatically remove the last dot (all unicode dots actually) in C++ depending on the setting.